### PR TITLE
dev-libs/capstone: don't install static libraries unconditionally

### PR DESCRIPTION
**Pull request by hamarituc**
Closes: https://bugs.gentoo.org/890941
Signed-off-by: Mario Haustein <mario.haustein@hrz.tu-chemnitz.de>

### CODE REVIEW

File: dev-libs/capstone/capstone-5.0_rc2-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+RDEPEND="python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}
+	python? ( dev-python/setuptools[${PYTHON_USEDEP}] )
+"
+BDEPEND="${DISTUTILS_DEPS}"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+distutils_enable_tests setup.py
+
+S=${WORKDIR}/${P/_rc/-rc}
```
Comment (ceamac) [inline dev-libs/capstone/capstone-5.0_rc2-r3.ebuild:37]: Please move `S` below `SRC_URI`

File: dev-libs/capstone/capstone-5.0_rc2-r3.ebuild [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	wrap_python ${FUNCNAME}
+}
+
+src_install() {
+	cmake_src_install
+
+	wrap_python ${FUNCNAME}
+
+	if ! use static-libs ; then
+		find "${D}" -name '*.a' -delete || die
```
Comment (ceamac) [inline dev-libs/capstone/capstone-5.0_rc2-r3.ebuild:94]:
```suggestion
		find "${ED}" -name '*.a' -delete || die
```

**Review by ceamac — CHANGES_REQUESTED** [review]

**Comment by ceamac** [comment]
I searched reverse dependencies that may need changing to use `static-libs`, I think only `app-emulation/qemu` qualifies.

File: dev-libs/capstone/capstone-5.0_rc2-r3.ebuild (hunk 2 above)
Comment (hamarituc) [reply dev-libs/capstone/capstone-5.0_rc2-r3.ebuild]: fixed

File: dev-libs/capstone/capstone-5.0_rc2-r3.ebuild (hunk 1 above)
Comment (hamarituc) [reply dev-libs/capstone/capstone-5.0_rc2-r3.ebuild]: fixed

**Review by ceamac — APPROVED** [review]
Will merge this later, with qemu

**Pull request closed without merging**